platform/pom.xml

```diff
@@ -27,6 +27,8 @@
     <packaging>pom</packaging>
     <properties>
         <joda-convert.version>1.7</joda-convert.version>
```

Comment:
✏️ I know this is out of scope for this PR but shouldn't all versions be defined in one place: the parent pom?

Reply:
Let's make sure we have a tech debt entry for it.